models/manticore-13b-chat-pyg-guanaco.json

```diff
@@ -0,0 +1,58 @@
+{
+  "_descriptorVersion": "0.0.1",
+  "datePublished": "2023-06-12T11:16:39.000Z",
+  "name": "Manticore 13B Chat Pyg Guanaco",
+  "description": "The model, augmented with Guanaco qLora, shows broad capabilities compared to other models like Wizard or Manticore. It excels in in-context learning and reasoning but may have weaknesses in coding. The model follows instructions, works as a chatbot, and produces intelligent responses. It accepts various prompting styles, including the ###-Variant. The model is generally unrestricted and doesn't berate users. Recommended settings include low temperature, low diversity, and slight repetition penalty.",
+  "author": {
+    "name": "Open Access AI Collective",
+    "url": "https://huggingface.co/openaccess-ai-collective/",
+    "blurb": "<blurb needed>"
```

Comment:
I know I have this in another place for `openaccess-ai-collective` but let's leave it empty `""` for this one